app/DAO/BakeryDB.scala

```diff
@@ -0,0 +1,98 @@
+package DAO
+
+import doobie._
+import doobie.implicits._
+import cats.effect.IO
+import doobie.postgres.implicits._
+import models.Product
+import cats.effect.unsafe.implicits.global
+import play.api.Configuration
+import play.api.db.Database
+import models.DBTransactor
+import java.time.LocalDateTime
+import java.util.UUID
+import javax.inject._
```

Comment:
please order your imports (at the very least all imports from the same top package together